targets/unity-v2/Testing/Tests/PersistentSockets/PubSubTest.cs

```diff
@@ -52,23 +52,10 @@ public override void Tick(UUnitTestContext testContext)
                 req.Events = new List<EventsModels.EventContents>();
                 req.Events.Add(ec);
 
-                PlayFabEventsAPI.WriteEvents(req, WriteEventSuccessful, WriteEventFail);
+                PlayFabEventsAPI.WriteEvents(req, null, null);
```

Comment:
Are you sure that a WriteEvents failure is correctly handled as "no-op"?
It might be relevant that a WriteEvents failure is just "test failure".

Reply:
WriteEvents failing may not be a direct indicator that PubSub is not working correctly. 

I don't think we should fail the test early for one WriteEvent failure. The test will fail at a timeout, but could potentially still get one successful WriteEvent.